saml.go

```diff
@@ -2,7 +2,7 @@ package saml2
 
 import dsig "github.com/russellhaering/goxmldsig"
 
-const issueInstantFormat = "2006-01-02T15:04:05"
+const issueInstantFormat = "2006-01-02T15:04:05Z"
```

Comment:
Good catch. The spec seems to eventually reference ISO 8601, which (with the addition of the Z) this matches exactly.

+1